### PR TITLE
added field for laserkey and laserkey-deposit in member specials

### PR DESCRIPTION
**Pull request by DMaendlen**
New laser has keys, people want to use the laser -> they need keys. This tracks

1. who has a key
2. has the person paid a deposit to ensure timely return of the key in case of leaving the space

Since we don't know if the deposit will stay at the current level (EUR 10,-) I put an IntegerField for the deposit instead of a boolean.

### CODE REVIEW

File: shackbureau/usermanagement/models.py [hunk 1]
```diff
@@ -513,6 +513,7 @@ class MemberSpecials(models.Model):
     member = models.OneToOneField("Member")
     has_matomat_key = models.BooleanField(default=False)
     has_snackomat_key = models.BooleanField(default=False)
+    has_laserkey = models.BooleanField(default=False)
```
Comment (rixx) [inline shackbureau/usermanagement/models.py]: `has_laser_key` would be better for consistent naming with `has_snackomat_key`

Comment (DMaendlen) [inline shackbureau/usermanagement/models.py]: You're right. New PR incoming.

**Comment by rixx** [comment]
Looks good to me - add the migration and maybe squash the commits afterwards, and I'm +1 for the merge.

**Comment by DMaendlen** [comment]
Commits squashed, migration not happening.

python manage.py makemigrations dies with 

> psycopg2.OperationalError: could not translate host name "db" to address: Name or service not known

and I don't have the time to work on this.

**Pull request merged**